app/src/main/java/mollie/tictactoe/activities/BoardActivity.java

```diff
@@ -0,0 +1,84 @@
+package mollie.tictactoe.activities;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+import mollie.tictactoe.GameHelper;
+import mollie.tictactoe.R;
+import mollie.tictactoe.StateManager;
+import mollie.tictactoe.ui.BoardView;
+import mollie.tictactoe.ui.CellButton;
+import mollie.tictactoe.ui.UIBoardManager;
+
+public class BoardActivity extends AppCompatActivity {
```

Comment:
@felipesere the naming here is Android convention, adding the suffix `Activity` to any class that extends Activity, mainly to avoid namespace conflicts I believe.  The role of the class, I suppose, is displaying a page to a user on which the user can act, and handling those actions, so providing a space for activity to take place.